exercises/simple-linked-list/instructions.md

```diff
@@ -0,0 +1,11 @@
+# Instructions
+
+Your task is to create a prototype for a playlist feature for a music player application.
+
+For the prototype version, create a singly linked list containing a range of numbers.
+
+Given a linked list, you should be able to
+- reverse the list
+- convert the list to an array
```

Comment:
I wonder if we could link this to "printing" the playlist or something.

Reply:
Yeah, if we could link it back to our use case that would help tremendously. I really felt like this convert-to-and-from-array thing came out of nowhere, but since it's what's in the spec we can't just arbitrarily ditch it, either.